src/main/java/Duke.java

```diff
@@ -1,10 +1,153 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    private static String logo  = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static ArrayList<Task> list;
+
+    public static void main(String[] args) throws DukeException {
+        list = new ArrayList<>();
```

Comment:
While this certainly works, it may be better to specify the type that this list would contain. Also, would it be better to have a TaskList class to keep track of Tasks?